test/unit/data-type.js

```diff
@@ -1081,7 +1081,7 @@ describe('SmallDateTime', function() {
       }, TypeError, 'Out of range.');
 
       assert.throws(() => {
-        TYPES.SmallDateTime.validate(new Date('June 1, 2079'));
+        TYPES.SmallDateTime.validate(new Date('June 7, 2079'));
```

Comment:
I think the pervious test : 
```
assert.throws(() => {
        TYPES.SmallDateTime.validate(new Date('May 7, 2079'));
      }, TypeError, 'Out of range.');
```
It is no longer needed. It will fail since May 7, 2079 is a valid smalldate  value.

Reply:
Perfect, thanks @MichaelSun90! Will circle back here in a couple hours after morning meetings.

Reply:
Thanks! 🙇

Reply:
All set, @MichaelSun90. Thanks again.